src/bin/cargo-miri.rs

```diff
@@ -301,17 +303,17 @@ fn in_cargo_miri() {
         Some("test") => (MiriCommand::Test, 3),
         Some("run") => (MiriCommand::Run, 3),
         Some("setup") => (MiriCommand::Setup, 3),
-        // Default command, if there is an option or nothing
+        // Default command, if there is an option or nothing.
```

Comment:
This is not a sentence, why should it be ended by a full stop? (Same goes for many other new full stops.)

I am fine with changing them to be sentences, but IMO such one-word commands being ended in a  full stop looks really weird.

Reply:
Again, I agree... but I thought I was just conforming to convention here.